core/manifest/ConfigStaticManifest.php

```diff
@@ -183,17 +183,15 @@ function getStatics() {
 	}
 
 	/**
-	 * Get the next token to process, incrementing the pointer
+	 * Get the next token to process, incrementing the pointer, skips any whitespace tokens
 	 *
-	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
 	 * @return null | mixed - Either the next token or null if there isn't one
 	 */
-	protected function next($ignoreWhitespace = true) {
```

Comment:
If someone is extending it, or if they pass in parameters to the method call (where none is expected) nothing is broken.

It's the inverse that is true; Extending methods WITHOUT the necessary parameters, or calling methods without required parameters included.